src/test/scala/com/cognite/sdk/scala/v1/ClientTest.scala

```diff
@@ -2,30 +2,47 @@ package com.cognite.sdk.scala.v1
 
 import java.net.UnknownHostException
 
-import cats.{Id, Monad}
+import cats.{Comonad, Id, Monad}
+import cats.effect._
 import com.cognite.sdk.scala.common.{ApiKeyAuth, Auth, CdpApiException, InvalidAuthentication, RetryingBackend, SdkTest}
 import com.softwaremill.sttp.Response
+import com.softwaremill.sttp.asynchttpclient.cats.AsyncHttpClientCatsBackend
 import com.softwaremill.sttp.testing.SttpBackendStub
 
+import scala.concurrent.ExecutionContext
+
 class ClientTest extends SdkTest {
   "Client" should "fetch the project using login/status if necessary" in {
     noException should be thrownBy new GenericClient(
       "scala-sdk-test")(
       implicitly[Monad[Id]],
+      implicitly[Comonad[Id]],
       auth,
       sttpBackend
     )
     new GenericClient("scala-sdk-test")(
       implicitly[Monad[Id]],
+      implicitly[Comonad[Id]],
       auth,
       sttpBackend
     ).projectName should not be empty
   }
+  it should "support async IO clients" in {
+    implicit val contextShift: ContextShift[IO] = IO.contextShift(ExecutionContext.global)
+    implicit val timer: Timer[IO] = IO.timer(ExecutionContext.global)
+    new GenericClient[IO, Nothing]("scala-sdk-test")(
+      implicitly[Monad[IO]],
+      implicitly[Comonad[IO]],
```

Comment:
Do we need to explicitly ask for these implicits here (and in other places)? Should they not be resolved... well.. implicitly?

Reply:
I think they should but I got some weird resolution error and since we're already doing this for `Monad[Id]` here I didn't bother looking into it more.